_includes/doc-registry.html

```diff
@@ -0,0 +1,31 @@
+{%- assign documentation_navigation=site.documentation_navigation -%}
```

Comment:
just curious, why the assignment? you can loop thru  `site.documentation_navigation` on the next line without assigning it.

Reply:
No idea.  Not my change, but is a 1:1 from the old site.  I'm updating to remove the assignment, but not actually sure where this template is used off the top of my head.